<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-3IC4HAG9" name="com.mitocode.controller.TestNG.TestTrabajadorController" tests="5" failures="0" timestamp="2020-08-07T09:48:46 COT" time="0.111" errors="2">
  <testcase name="testRegistrarTrabajador" time="0.064" classname="com.mitocode.controller.TestNG.TestTrabajadorController">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.mitocode.controller.TrabajadorController.registrar(TrabajadorController.java:70)
at com.mitocode.controller.TrabajadorController$$FastClassBySpringCGLIB$$a1d7619f.invoke(<generated>)
at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:749)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:688)
at com.mitocode.controller.TrabajadorController$$EnhancerBySpringCGLIB$$4ce354a5.registrar(<generated>)
at com.mitocode.controller.TestNG.TestTrabajadorController.testRegistrarTrabajador(TestTrabajadorController.java:34)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
]]>
    </error>
  </testcase> <!-- testRegistrarTrabajador -->
  <system-out/>
  <testcase name="testModificarTrabajador" time="0.001" classname="com.mitocode.controller.TestNG.TestTrabajadorController"/>
  <system-out/>
  <testcase name="testListarPorEmpresaTrabajador" time="0.004" classname="com.mitocode.controller.TestNG.TestTrabajadorController">
    <error type="org.springframework.security.authentication.AuthenticationCredentialsNotFoundException" message="An Authentication object was not found in the SecurityContext">
      <![CDATA[org.springframework.security.authentication.AuthenticationCredentialsNotFoundException: An Authentication object was not found in the SecurityContext
at org.springframework.security.access.intercept.AbstractSecurityInterceptor.credentialsNotFound(AbstractSecurityInterceptor.java:379)
at org.springframework.security.access.intercept.AbstractSecurityInterceptor.beforeInvocation(AbstractSecurityInterceptor.java:223)
at org.springframework.security.access.intercept.aopalliance.MethodSecurityInterceptor.invoke(MethodSecurityInterceptor.java:65)
at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:688)
at com.mitocode.controller.TrabajadorController$$EnhancerBySpringCGLIB$$4ce354a5.listarPorEmpresa(<generated>)
at com.mitocode.controller.TestNG.TestTrabajadorController.testListarPorEmpresaTrabajador(TestTrabajadorController.java:59)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
]]>
    </error>
  </testcase> <!-- testListarPorEmpresaTrabajador -->
  <system-out/>
  <testcase name="CrearContratoTrabajador" time="0.035" classname="com.mitocode.controller.TestNG.TestTrabajadorController"/>
  <system-out/>
  <testcase name="CrearTrabTrabajador" time="0.007" classname="com.mitocode.controller.TestNG.TestTrabajadorController"/>
  <system-out/>
</testsuite> <!-- com.mitocode.controller.TestNG.TestTrabajadorController -->
